symplyphysics/symbols/classical_mechanics.py

```diff
@@ -75,6 +75,11 @@
 **Area** is the size of a region on a two-dimensional surface.
 """
 
+arc_length = SymbolNew("s", units.length)
```

Comment:
Maybe write it as `arc_length = distance` ?

Reply:
I think we need to distinguish between distance as the length of the smallest path connecting two points, i.e. the length of the segment of a straight line connecting them or equivalently the Euclidean distance, and distance as the overall length of the path of a body during its travel. These are two very different definitions of distance from a physical standpoint, but unfortunately we tend to conflate the two in the language we use to describe physical events. That's why I'm trying to be clear about what the symbols we use in laws actually represent.

`arc_length` belongs to the second definition of distance I've written above. Specifically, it's the length of a part of a curve, which is the same as the Euclidean distance iff the curve is a straight line, so they're not equal in general.

Reply:
Add small note to the comments, so we won't try to fix it later.